NEWS.md

```diff
@@ -38,14 +38,15 @@
 ### Posit Workbench
 
 - Adds `-l` (long) option to `rserver-url`. When `/usr/lib/rstudio-server/bin/rserver-url -l <port number>` is executed within a VS Code or Jupyter session, the full URL where a user can view a server proxied at that port is displayed (rstudio-pro#3620)
-- Handles `SIGTERM` signals to shut down more gracefully, for better interaction with service managers like `systemd`.
+- Handles `SIGTERM` signals to shut down more gracefully, for better interaction with service managers like `systemd`. (rstudio/rstudio-pro#2585)
```

Comment:
```suggestion
```
I think we should remove this, as you mentioned it's redundant to the new entires in the fixed section.